esp/platform/espcontext.hpp

```diff
@@ -46,6 +46,10 @@ static const char* const SESSION_AUTH_MSG_COOKIE = "ESPAuthenticationMSG";
 static const char* const DEFAULT_LOGIN_URL = "/esp/files/Login.html";
 static const char* const DEFAULT_GET_USER_NAME_URL = "/esp/files/GetUserName.html";
 static const char* const DEFAULT_UNRESTRICTED_RESOURCES = "/favicon.ico,/esp/files/*,/esp/xslt/*";
+static const char* const AUTH_STATUS_NA = "NA";
+static const char* const AUTH_STATUS_FAIL = "Fail";
+static const char* const AUTH_STATUS_OK = "Ok";
+static const char* const AUTH_STATUS_NOACCESS = "NoAccess"; //failed for feature level authorization
```

Comment:
Curious why was it decided to pass strings to the set function instead of passing a type safe enumerated type? You could then have a get(enum) method that returns the enum, and another one that returns these strings?

Reply:
It shows inside esp log. Easy to read and not too many characters.